### PR TITLE
Vtcca 3203

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala [hunk 1]
```diff
@@ -39,6 +40,23 @@ class ExternalPropertyLinkApi @Inject()(
 )(implicit executionContext: ExecutionContext)
     extends BaseVoaConnector {
 
+  def getMyAgentPropertyLinks(
+        agentCode: Long,
+        searchParams: GetMyOrganisationPropertyLinksParameters,
+        params: Option[PaginationParams])(implicit request: RequestWithPrincipal[_]): Future[PropertyLinksWithAgents] =
```
Comment (DanielConnelly) [inline app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala]: why is paragination parameters optional?

Comment (reddy-yattapu) [reply app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala]: removed Option for pagiantion

**Comment by jon-pike** [comment]
👍

**Comment by habeeb01** [comment]
LGTM 👍

**Pull request merged**